src/cpp/configuration/implementations/DiscoveryServerConnectionAddress_configuration.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file DiscoveryServerConnectionAddress_configuration.cpp
+ *
+ */
+
+#include <ddsrouter/exceptions/ConfigurationException.hpp>
+#include <ddsrouter/exceptions/UnsupportedException.hpp>
+#include <ddsrouter/types/address/Address.hpp>
+#include <ddsrouter/types/address/DiscoveryServerConnectionAddress.hpp>
+#include <ddsrouter/types/configuration_tags.hpp>
+#include <ddsrouter/types/RawConfiguration.hpp>
+
+namespace eprosima {
+namespace ddsrouter {
+
+DiscoveryServerConnectionAddress::DiscoveryServerConnectionAddress(
```

Comment:
If you have to import the yaml header file in all configuration files anyway, why aren't the implementations of these functions in their corresponding implementation files?
This applies to `Addresss_configuration.cpp`, `DiscoveryServerConnectionAddress_configuration.cpp`, `DomainId_configuration.cpp`, and `GuidPrefix_configuration.cpp`.

Reply:
Already discussed, we'll leave it this way and change it in the future.